src/cmd/networkmapper/list/mapper-list.go

```diff
@@ -19,9 +21,11 @@ var ListCmd = &cobra.Command{
 				return err
 			}
 			if viper.IsSet(mapperclient.MapperExcludeServices) {
-				intents = mappershared.RemoveExcludedServices(intents, nil)
+				intents = mappershared.RemoveExcludedServices(intents, viper.GetStringSlice(mapperclient.MapperExcludeServices))
+			}
+			if viper.Get(config.OutputFormatKey) != config.OutputFormatDefault {
+				logrus.Warn("Different output formats are not supported in 'network-mapper list' command;")
```

Comment:
Let's turn this into an error? And suggest the right thing to do - use `export` instead